--- Makefile	2015-11-15 03:53:29.000000000 -0500
+++ Makefile	2015-11-15 03:44:18.000000000 -0500
@@ -183,4 +183,20 @@
 	_usertests\
 	_wc\
 	_zombie\
+	_crowd_can_test\
+	_crowd_childb4parent\
+	_crowd_parentb4child\
+	_crowd_yieldother\
+	_crowd_gettid\
+	_crowd_single_thread_texit_exit\
+	_crowd_basic_mkrun_desch\
+	_crowd_yield_lots\
+	_crowd_killfake\
+	_crowd_maxspawnfork\
+	_crowd_outofthreads\
+	_crowd_wildpointer\
+	_crowd_yieldsame\
+	_crowd_check_tspawn_bounds\
+	_crowd_check_desch_bounds\
+	_crowd_tspawn_texit_stress_test\

